core/DataAccess/ArchiveWriter.php

```diff
@@ -171,6 +171,7 @@ private function getModel()
     protected function logArchiveStatusAsIncomplete()
     {
         $this->insertRecord($this->doneFlag, self::DONE_ERROR);
+        $this->flushSpool('numeric');
```

Comment:
btw I had removed that one before merging the PR re the spools as after some more investigating found it wasn't needed.